setup.py

```diff
@@ -18,8 +18,15 @@
 # along with ceph_exporter.  If not, see
 # <http://www.gnu.org/licenses/>.
 
+import re
+import platform
 from setuptools import setup
 
+def data_files():
+    if platform.system() == 'Linux':
+      if platform.linux_distribution()[0] == 'CentOS Linux':
+        return ('/lib/systemd/system', ['ceph_exporter.service'])
+        
```

Comment:
Service files should only be installed in /lib/systemd/system if they are installed using a package manager. Otherwise they need to go into /etc/systemd/system. Also, why limit this to CentOS? Many Linux distros are using systemd now so there's probably a better way to do this.